doc/shared/legal.rst

```diff
@@ -1,6 +1,5 @@
 .. _fwk_rtos_copyright:
 
-.. include:: ../_templates/disclaimer.rst
```

Comment:
Why has the disclaimer been removed?

Reply:
@BrennanGit, can you answer the question above?

Reply:
These days we don't autogenerate a disclaimer, in general moving away from autogenerated pages.
The alternative is to use a set of substitutions like we see in fwk_core: https://raw.githubusercontent.com/xmos/fwk_core/develop/doc/shared/legal.rst

Not sure why it wasn't added here, happy to do so if required.

Reply:
As with fwk_core, we should include in fwk_rtos (and fwk_io if it's not already there):

```
Copyright & Disclaimer
======================

|XMOS copyright|

|XMOS disclaimer|

|XMOS trademarks|

```

This text appears above the Licence section.